packages/terra-html-table/src/TableHeader.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
+};
+
+const TableHeader = ({
+  children,
+  ...customProps
+}) => (
+  <thead {...customProps}>
+    <tr>
```

Comment:
Would there be any reason for consumers to need to add attributes to the table-header row?